f3/granite.go

```diff
@@ -391,15 +391,10 @@ func (i *instance) VerifyJustification(justification Justification) error {
 
 	payload := SignaturePayload(justification.Payload.Instance, justification.Payload.Round, justification.Payload.Step, justification.Payload.Value)
 	signers := make([]PubKey, 0)
-	if err := justification.QuorumSignature.Signers.ForEach(func(bit uint64) error {
-		if int(bit) >= len(i.powerTable.Entries) {
-			return fmt.Errorf("invalid signer index: %d", bit)
-		}
+	_ = justification.QuorumSignature.Signers.ForEach(func(bit uint64) error {
```

Comment:
Ignoring an error raises a big flag while reading the code, so please add a comment explaining why.

"Note that indexing the power table entries with an invalid index will panic here, but the index was checked earlier. There's no other error returned".

This raises a bigger question though: why are we iterating Signers twice? Can you collapse this to a single iteration?

Reply:
I can collapse it, but thought of separating conceptually different checks on different iterations (and also exit quickly without any extra work if the message does not contain a strong quorum). But actually you are right, not like spamming with certificates not containing a strong quorum in the list of signers will change much (if the sender wants to spam, that is). Pushed the conflated iteration.